extensions/wikia/AdEngine/js/video/player/ui/videoInterface.js

```diff
@@ -0,0 +1,22 @@
+/*global define, require*/
+define('ext.wikia.adEngine.video.player.ui.videoInterface', [
+	'wikia.log'
+], function (log) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.video.player.ui.videoInterface';
+
+	function setup(video, uiElements, params) {
+		uiElements.forEach(function (element) {
+			var module = 'ext.wikia.adEngine.video.player.ui.' + element;
```

Comment:
well we can have defined object with all the possible modules and then we can take a module name from that object. This has one disadvantage, when adding new UI module you have to remember to add it to both AssetsManager/config.php and to this object.

On the other hand having a comment with names above forces you to remember about this place as well.